.github/workflows/format.yaml

```diff
@@ -0,0 +1,30 @@
+on:
+  push:
+    branches: master
+  pull_request:
+    branches:
+      - master
+
+name: format_check
+
+jobs:
+  format_check:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install ClangFormat
+        run: sudo apt-get install -y clang-format-10
+
+      - name: Run ClangFormat
+        run: make format clang_format=clang-format-10
+
+      - name: Check for a non-empty diff
+        run: |
+          if [ "$(git diff --name-only)" != "" ]
```

Comment:
I have used `git diff-files -U --exit-code` for this in the past (https://github.com/tidyverse/readr/blob/master/travis/check_format.sh#L1-L13)

I think it is a bit safer than checking for empty names, though probably what you have is fine in practice